cpp/tests/types/type_dispatcher_test.cu

```diff
@@ -84,4 +84,95 @@ TEST_P(IdDispatcherTest, IdToType)
   EXPECT_TRUE(cudf::type_dispatcher(cudf::data_type{t}, verify_dispatched_type{}, t));
 }
 
+template <typename T>
+struct DoubleTypedDispatcherTest : public DispatcherTest {
```

Comment:
```suggestion
struct DoubleTypeDispatcherTest : public DispatcherTest {
```

Reply:
@harrism I was matching the name of the existing `TypedDispatcherTest` but I agree this is a better name. May I change the single-dispatch test names as well?

Reply:
It's not a big deal. Up to you.

Reply:
Don't change the names. TypedDispatcherTest reflects the fact that it is the _typed_ fixture class for the type-parameterized tests.

Reply:
As such, the extension of naming for the double dispatcher would be `TypedDoubleDispatcherTest`.

Reply:
Good catch @jrhemstad. Fixed in 748124b7fe786fda6ecacf5d1b4beebc98271ef9.